src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './components';
+export { UIToolkit } from './styles/UIToolkit'
```

Comment:
this is still a component, I'd move it under HOC directory in components

Reply:
but let's align first on the solution before applying this

Reply:
Updated after discussion.